app-backend/common/src/main/scala/CommonHandlers.scala

```diff
@@ -0,0 +1,14 @@
+package com.azavea.rf.common
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.StandardRoute
+import akka.http.scaladsl.server.directives.RouteDirectives
+
+
+trait CommonHandlers extends RouteDirectives {
```

Comment:
The common thread I see between this and `UserErrorHandler` is that they both interpret various types of reporting from the backend and convert it into HTTP status codes for the client. So in that respect I think it might make sense to roll them together, but maybe with a more descriptive name that reflects the common functionality.

Another approach would be to leave them separate. I think that's defensible too: `UserErrorHandler` deals with raised exceptions whereas this interprets the results of queries that were successful, so they're operating under different expected conditions. If you choose to go that route I think it might make sense to write a convenience wrapper to reduce the amount of boilerplate necessary to use both at the same time.